flightros/src/motion_planning/motion_planning.cpp

```diff
@@ -1,15 +1,36 @@
 #include "flightros/motion_planning/motion_planning.hpp"
 
-#define CONTROL_UPDATE_RATE 50.0
-
-#define TINYPLY_IMPLEMENTATION
-
 namespace ob = ompl::base;
 namespace og = ompl::geometric;
+
+using namespace flightlib;
 using namespace tinyply;
 
-std::vector<motion_planning::float3> motion_planning::readPointCloud() {
-  std::unique_ptr<std::istream> file_stream;
+namespace motion_planning {
+
+MotionPlanner::MotionPlanner() {}
+
+MotionPlanner::~MotionPlanner() {}
+
+void MotionPlanner::run() {
+  std::cout << "Read PointCloud" << std::endl;
+  readPointCloud();
```

Comment:
what is readPointCloud() failed. e.g., there is no point cloud file? would be nice to print hint messages in case it does not work.

Reply:
This function can basically encounter two errors. First is a runtime error when there is no point cloud file. Another possibility is that tiniyply encounters an exception because the file is not properly formatted. The error messages can be seen on line 42, 81 and 117. Any other information that should be printed out?